### PR TITLE
Add IntSelect<>

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: functions/int_select.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#ifndef FUNCTIONS_INT_SELECT_H
+#define FUNCTIONS_INT_SELECT_H
+
+// Usage: IntSelect<SELECTION, N>
```
Comment (profezzorn) [inline functions/int_select.h:4]: This needs to show that you can have as many integers as you like.

Comment (fdarosa2663) [reply functions/int_select.h]: updated

File: functions/int_select.h [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        case 32768: return FunctionRunResult::ONE_UNTIL_IGNITION;
+        default: break;
+      }
+    }
+    return FunctionRunResult::UNKNOWN;
+  }
+  int getInteger(int led) { return value_; }
+
+private:
+	F f_;
```
[the remark this answers is not in the document]
Comment (fdarosa2663) [reply functions/int_select.h]: fixed

File: functions/int_select.h [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#ifndef FUNCTIONS_INT_SELECT_H
+#define FUNCTIONS_INT_SELECT_H
+
+// Usage: IntSelect<SELECTION, Int1, Int2...>
+// SELECTION: FUNCTION
+// Returns SELECTION of N
```
Comment (profezzorn) [inline functions/int_select.h:6]: This description still needs work.
An example could work, like say, if SELECTION is 0, the first integer will be returned, if SELECTION is 1, the second, and so forth.

Comment (fdarosa2663) [reply functions/int_select.h]: updated

**Pull request merged**